src/gRPC/GRPCServer.ts

```diff
@@ -77,6 +84,49 @@ export class GRPCServer {
             callback: grpc.sendUnaryData<RegisterAntennaResponse>
         ) => this.registerAntennaRoute(call, callback),
     };
+    getantennaRoute(
+        call: grpc.ServerUnaryCall<Empty__Output, GetAntennasResponse>,
+        callback: grpc.sendUnaryData<GetAntennasResponse>
+    ): void {
+        this.db.getAllAntennas().then((antenna) => {
+            let response = this.convertAntennaObjectToGetAntennasResponse(antenna);
+            if (response.status == 'fail') {
```

Comment:
If `convertAntennaObjectToGetAntennasResponse` is only used here, why not make it return an object that is easier to use in the `callback` for example, use the grpc status codes for the response status.